src/titiler/application/titiler/application/settings.py

```diff
@@ -9,6 +10,7 @@ class ApiSettings(pydantic.BaseSettings):
     name: str = "titiler"
     cors_origins: str = "*"
     cachecontrol: str = "public, max-age=3600"
+    root_path: str = os.getenv("ROOT_PATH", "")
```

Comment:
you don't need to use os.getenv here (see https://pydantic-docs.helpmanual.io/usage/settings/#environment-variable-names) 

this means that to set `root_path` you'll need to have `TITILER_API_ROOT_PATH` either in your environment or in a `.env` file

Reply:
Oh! Cool, ok, will change. Thanks for the help!